src/fmu/dataio/_definitions.py

```diff
@@ -57,6 +57,7 @@ def __post_init__(self):
     "volumetrics": None,  # or?
     "khproduct": None,
     "timeseries": None,
+    "unsmry": None,
 }
```

Comment:
Perhaps we have discussed this before, but all words here are generic and human readable, while `unsmry` refers to a spesific format `unified summary` files (there are `non-unified summary` files also; was actually the normal "best practice" prior to 2005'ish) made by Eclipse and later by other simulators (sometimes as workaround, e.g. IX). Isn't summary files just time series?

Reply:
Suggestion ref offline discussions:

```
data.content: flowsummary
  flowsummary:
    simulator: eclipse
```